catroid/src/main/java/org/catrobat/catroid/pocketmusic/ui/TactScrollRecyclerView.java

```diff
@@ -52,11 +52,14 @@
 	private static final int MINIMUM_TACT_COUNT = 2;
 	private static final int TACT_VIEW_TYPE = 0;
 	private static final int PLUS_VIEW_TYPE = 1;
+	private static final int EMPTY_VIEW_TYPE = 2;
```

Comment:
I would assign TACTS_PER_SCREEN to it. Maybe a more specific name would be good: EMPTY_PLAY_ON_END_VIEW_TYPE?

Reply:
Changed the name and the value of the types to keep the confusion to a minimum.